src/Http/Http.Abstractions/src/RouteHandlerInvocationContext.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.AspNetCore.Http;
 /// Provides an abstraction for wrapping the <see cref="HttpContext"/> and parameters
 /// provided to a route handler.
 /// </summary>
-public class RouteHandlerInvocationContext
+public sealed class RouteHandlerInvocationContext
```

Comment:
Don't seal this one. We'll end up un sealing it.

Reply:
We discussed this during API review. Unsealing doesn't have a cost, but sealing does so we are taking the more flexible option to start.

If we ship the parameters change in preview4, it shouldn't be too bad.